funnel/assets/sass/_layout.scss

```diff
@@ -957,6 +957,38 @@
   height: 99% !important;
 }
 
+:not(pre) > code,
```

Comment:
Doesn't the `:not()` selector have a significant performance impact?

Reply:
Apparently not a concern anymore. I remember when it was introduced and was controversial for the performance impact it caused. https://stackoverflow.com/questions/44668606/css-not-performance